examples/working-directly-with-json/working_directly_with_json.out

```diff
@@ -1,3 +1,3 @@
-bal run working_directly_with_json.bal
+$ bal run working_directly_with_json.bal
```

Comment:
`$` was removed after some discussions IIRC. Shall we finalize if we are adding it back before changing it here?

Reply:
@anupama-pathirage @sameerajayasoma

Reply:
I prefer having it too btw. And as discussed, since with BBEs the user may not generally have the same file name, it's highly unlikely they'll copy the command as is. So copying shouldn't be an issue here IMO. We'll also have the output in the same file which is probably why we don't give the option to copy atm?

Reply:
What if we add it later, when we figure out things (like having a way to copy without the `$` sign etc)

Reply:
Thanks, Yasith! Will keep the $sign as we will be implementing this fix soon.

Reply:
I was just thinking maybe @keizer619's comment about reverting (deleting the description file) in a single PR would apply here too? Should we do this change for all BBEs also in a single PR?

Reply:
We have already merged the fix of this and now the $sign isn't getting copied. Therefore, we can finalize adding this.